ironmq/src/main/scala/akka/stream/alpakka/ironmq/CommittableMessage.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.ironmq
+
+import akka.Done
+
+import scala.concurrent.Future
+
+/**
+ * Commit an offset that is included in a [[CommittableMessage]].
+ */
+trait Committable {
+  def commit(): Future[Done]
```

Comment:
We can not force Java people to do extra work, our APIs need to be just as good for both languages.

I suggest doing `ironmq/src/main/scala/akka/stream/alpakka/ironmq/javadsl/CommittableMessage.scala`
and `ironmq/src/main/scala/akka/stream/alpakka/ironmq/scaladsl/CommittableMessage.scala`

as other components in Alpakka and Akka Streams itself does.

You can get away without implementing the stage twice - simply in each javadsl/scaladsl you'd call the same `Graph[...]` (the actual impl).